src/espIdf/serial/serialPort.ts

```diff
@@ -71,12 +71,14 @@ export class SerialPort {
   }
 
   private async updatePortListStatus(l: string) {
-    await idfConf.writeParameter("idf.port", l);
-    const portHasBeenSelectedMsg = this.locDic.localize(
-      "serial.portHasBeenSelectedMessage",
-      "Port has been updated to "
-    );
-    Logger.infoNotify(portHasBeenSelectedMsg + l);
+    await idfConf.chooseConfigurationTarget().then(async (target) => {
+      await idfConf.writeParameter("idf.port", l, target);
```

Comment:
lets just use `async/await` and not mixin with `.then/.catch` block

The above code can be re-written like this which makes it easier to read and does the same thing

```js
const target = await idfConf.chooseConfigurationTarget();
await idfConf.writeParameter("idf.port", l, target);
```